src/libraries/System.Reflection.DispatchProxy/ref/System.Reflection.DispatchProxy.csproj

```diff
@@ -1,6 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>$(NetCoreAppCurrent);netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent);netstandard2.0;net461;$(NetFrameworkCurrent)</TargetFrameworks>
```

Comment:
I think you can just delete this package per https://github.com/dotnet/runtime/issues/34253#issuecomment-615441085.

Reply:
While I'm fine with that, should we do that in a different PR? The reason behind it being that we may want to revert this in case it breakes something or alternatively revert the removal of the package for dispatchProxy, and I wouldn't want that to mean that we have to revert both. Also, conceptually they are different.

Reply:
Fair, we can come back and delete the package and while doing so clean up the configurations.

Reply:
So it looks like you're now adding NetCoreApp to the package?  If this thing is causing trouble I'd really prefer we just stop shipping it.  There isn't a good reason to ship this thing OOB now that the API is fixed by being part of `netstandard2.1`.  I'll submit a PR you can either choose to rebase on that or pull it into this one.

Reply:
I added netcoreapp in order to fix the issue and cycle caused by not having it. That said, I'm fine to remove the package if that's what we want, I'm happy to do it as part of this PR as well.

Reply:
This is done now in my last commit, please take a look to make sure I didn't miss anything.